web/src/main/java/org/openmrs/web/Listener.java

```diff
@@ -426,14 +427,16 @@ private void copyCustomizationIntoWebapp(ServletContext servletContext, Properti
 					webAppLog.debug("Overriding file: " + absolutePath);
 					webAppLog.debug("Overriding file with: " + userOverridePath);
 					if (file.isDirectory()) {
-						for (File f : file.listFiles()) {
-							userOverridePath = f.getAbsolutePath();
-							if (!f.getName().startsWith(".")) {
-								String tmpAbsolutePath = absolutePath + "/" + f.getName();
-								if (!copyFile(userOverridePath, tmpAbsolutePath)) {
-									webAppLog.warn("Unable to copy file in folder defined by runtime property: " + prop);
-									webAppLog.warn("Your source directory (or a file in it) '" + userOverridePath
-									        + " cannot be loaded or destination '" + tmpAbsolutePath + "' cannot be found");
+						if (file.listFiles() != null) {
+							for (File f : Objects.requireNonNull(file.listFiles())) {
```

Comment:
Do you need Objects.requireNonNull when you are already sure that folder.listFiles() is not null?